test/api_callable.js

```diff
@@ -378,4 +378,35 @@ describe('retryable', function() {
       done();
     });
   });
+
+  it('reports A/B testing', function() {
```

Comment:
We need to disable the A/B testing stuff for now. The service control team does not want to see this in our headers, so I need to figure out another approach to report this data (likely using another header).